setup.py

```diff
@@ -0,0 +1,16 @@
+import setuptools
+
+
```

Comment:
To generate the version file when building the conda package do add this:

```
scm_version_template = """# Generated by setuptools_scm
__all__ = ["__version__"]

__version__ = "{version}"
"""
```

here then...